terraspace/app/stacks/publishing/tfvars/base.tfvars

```diff
@@ -0,0 +1,22 @@
+provider_ads_bucket_name = "<%= expansion(':ENV') %>-ep-ipfs-advertisement"
+content_lambda = {
+  name              = "<%= expansion(':ENV') %>-ep-publishing-content"
+  metrics_namespace = "<%= expansion(':ENV') %>-ep-publishing-content"
+  image_uri         = "505595374361.dkr.ecr.us-west-2.amazonaws.com/publisher-lambda:latest"
+}
+ads_lambda = {
+  name              = "<%= expansion(':ENV') %>-ep-publishing-advertisement"
+  metrics_namespace = "<%= expansion(':ENV') %>-ep-publishing-advertisement"
+  image_uri         = "505595374361.dkr.ecr.us-west-2.amazonaws.com/publisher-lambda:latest"
```

Comment:
can we avoid hardcoding the registry?

Reply:
Done! Made it consistent with the indexing stack.